dlrs/main/classes/RollupServiceTest.cls

```diff
@@ -1062,12 +1064,14 @@ private with sharing class RollupServiceTest {
     // + One query for the Account query above
     // + One query on Rollup object
     // + One query on Opportunity for rollup a
-    System.assertEquals(9, Limits.getQueries());
+    // + One query on Parent for values to confirm DML needed
+    System.assertEquals(11, Limits.getQueries());
```

Comment:
This one doesn't add up, so I am confused, why we need 2 more queries than before.

Reply:
7 increased to 8 on line 1046 which means line 1068 now has a base +1 on it's math, then we added another query which is another +1, totaling in +2 being added though only one was caused by line 1062.